compiler/aarch64/codegen/OMRMachine.cpp

```diff
@@ -97,13 +207,31 @@ TR::RealRegister *OMR::ARM64::Machine::freeBestRegister(TR::Instruction *current
    int32_t dataSize = 0;
    TR::InstOpCode::Mnemonic loadOp;
 
+   cg->traceRegisterAssignment("FREE BEST REGISTER FOR %R", virtualRegister);
+
    if (forced != NULL)
       {
       best = forced;
       candidates[0] = best->getAssignedRegister();
       }
    else
       {
+      uint32_t interference = 0; // TODO implement live register analysis for aarch64
+      uint32_t preference = 0, pref_favored = 0;
```

Comment:
I think it is better to declare `pref_favored` as a variable of type `bool`.

Reply:
Changed `pref_favored` type to `bool`.